src/poliastro/plotting/_base.py

```diff
@@ -170,7 +170,7 @@ def _plot_maneuver(
 
         if len(maneuver_phases) == 0:
             # For single-impulse maneuver only draw the impulse marker
-            self._draw_impulse(color, f"Impulse - {label}", maneuver_phases[0].r)
+            return ([self._draw_impulse(color, f"Impulse - {label}", final_phase.r)],)
```

Comment:
Right, we need a `return`, otherwise the `coordinates = concatenate_representations(coordinates_list)` line below fails.

Reply:
Also, if there is only one impulse, there is no line to draw.